app/models.py

```diff
@@ -109,6 +109,35 @@ class DocumentRecommendation(db.Model):
     )
 
 
+class ModelSteps(db.Model):
```

Comment:
```suggestion
class ModelStep(db.Model):
```
Alle Tabellen sind im singluar